### PR TITLE
[Cl] Slack Workflow

### PR DESCRIPTION
**Pull request by theBeginner86**
Signed-off-by: Pranav Singh <pranavsingh02@hotmail.com>

**Description**
Fix working of slack workflow

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ jobs:
...
-        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/discuss-board-topics' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/discuss-twitter' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred discuss-board-topics! (https://github.com/layer5io/discuss-board-topics/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"${{ github.actor }} just starred discuss-twitter repo! (https://github.com/layer5io/discuss-twitter/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```
Comment (warunicorn19) [inline .github/workflows/slack.yml:16]: @theBeginner86 just one nitpick 😅, let's just keep it as "starred discuss-twitter!" No need to write "repo" just to maintain consistency.

Comment (theBeginner86) [reply .github/workflows/slack.yml]: Gotcha 😁

**Comment by theBeginner86** [comment]
@warunicorn19
I've removed that extra word!!

**Review by warunicorn19 — APPROVED** [review]
Thank you @theBeginner86 😁

**Comment by leecalcote** [comment]
How that other portion for repo name coming along, @theBeginner86? 😁

**Comment by theBeginner86** [comment]
> How that other portion for repo name coming along, @theBeginner86? 😁

@leecalcote 
Actually, I thought that it was a typo and was causing the problem. So, I changed with the one found in the URL of this repo which has `/discuss-twitter`.
Did I make a mistake there? 😅

**Comment by warunicorn19** [comment]
@theBeginner86 i think @leecalcote is talking about something else 😅(maybe). It was indeed the repo-name that was causing the issue.

**Comment by theBeginner86** [comment]
I'm working on that for this issue: https://github.com/meshery/meshery-istio/issues/314
Once it works properly over there then we can swap it on every repo 😁

**Pull request merged**